src/assets/styles/components/buttons.scss

```diff
@@ -0,0 +1,5 @@
+.btn {
+  text-transform: uppercase;
```

Comment:
all buttons to have uppercase text? [usually uppercase text has lower legibility](https://blog.prototypr.io/all-caps-on-ui-good-or-bad-2570f14dc457)